missilewars-plugin/src/main/java/de/butzlabben/missilewars/game/GameManager.java

```diff
@@ -59,7 +59,7 @@ public void loadGamesOnStartup() {
         if (Config.isMultipleLobbies()) {
             lobbyFiles = new File(Config.getLobbiesFolder()).listFiles();
         } else {
-            File file = new File(Config.getLobbiesFolder() + "/" + Config.getDefaultLobby());
```

Comment:
We should leave this in here, as java can parse it with to slashes, e.g. "folder1//folder2" so that the admin does not have to worry about that. A more elegant solution would be to using the subdirectory constructor of File 
```java
File lobbiesFolder = new File(Config.getLobbiesFolder());
File file = new File(lobbiesFolder, Config.getDefaultLobby());
```

Reply:
Done.